data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/IdsMultipartSender.java

```diff
@@ -101,7 +91,7 @@ private static URI createConnectorIdUri(String connectorId) {
      */
     @Override
     public CompletableFuture<R> send(M request, MessageContext context) {
```

Comment:
the `MessageContext` is not used, may worth removing it to keep the interface simpler

Reply:
done